01-Login/templates/assets/style.css

```diff
@@ -2,16 +2,22 @@
   padding: 20px 30px;
   display: flex;
   align-items: center;
-  background-color: #27417E;
+  background-color: #282727;
   color: white;
 }
+
 .header-text {
   font-size: 32px;
   margin-left: 10px;
 }
+
+.header-link {
+  color: white;
+  text-decoration-color: white;
+}
   
 .passage-logo {
-  background-image: url('https://storage.googleapis.com/passage-docs/passage-logo.svg');
+  background-image: url('https://storage.googleapis.com/passage-docs/passage-logo-dark.svg');
```

Comment:
Is this logo and the other background-image logo that was changed meant to be just the logo icon or should it include the 'Passage' text next to it like in https://storage.googleapis.com/passage-docs/passage-logo-dark-full.png?

Reply:
yes we just use the logo icon without the Passage text in the examples